fargate/upload-move/store.go

```diff
@@ -182,6 +182,17 @@ func (s *UploadMoveStore) moveFile(workerId int, timeout time.Duration, items <-
 			continue
 		}
 
+		// Now that we know the storage bucket, make a new client with the correct region
+		var region string
```

Comment:
do you need this var? I would think you could just have `s.s3, region, err :=` below?